src/cms/media-gallery-asset-management.md

```diff
@@ -39,18 +39,18 @@ Assets can be used extensively throughout the Admin, such as [add or edit a page
     ![]({% link images/images/enhanced-media-gallery/selected-asset.png %}){: .zoom}
     _Add Selected_
 
-## Delete an asset
+## Delete assets
 
 1. On the _Admin_ sidebar, go to **Content** > _Media_ > **Media Gallery**.
 
-1. Select the asset, then click <span class="btn">Delete Selected</span>.
+1. Click <span class="btn">Delete Images...</span> and use checkboxes to select assets that need to be deleted.
 
     ![]({% link images/images/enhanced-media-gallery/selected-asset.png %}){: .zoom}
```

Comment:
It looks like the PR deletes this file -- did you want to include a replacement?

Reply:
Yes, it is included.
<img width="1270" alt="Screen Shot 2020-07-28 at 3 37 35 PM" src="https://user-images.githubusercontent.com/17548019/88719079-54af8300-d0e8-11ea-818c-46820be56013.png">